integration/testdata/generate_pipeline/existing_other/expectedSkaffold.yaml

```diff
@@ -14,6 +14,7 @@ profiles:
     - image: gcr.io/first-week-244218/getting-started-pipeline
       context: .
       kaniko: {}
+    debugHelpersRegistry: gcr.io/gcp-dev-tools/duct-tape
```

Comment:
Through my other changes the default YAML contained this property. The whole test in this form shouldn't be necessary anymore because I moved the property to the global config